packages/circuit-ui/components/TextArea/useAutoExpand.spec.tsx

```diff
@@ -157,29 +159,29 @@ describe('useAutoExpand hook', () => {
       expect(ref.current).toHaveAttribute('style');
     });
 
-    test('should modified provided `onInput`', () => {
+    test('should modify provided `onInput`', async () => {
       const ref = createTextAreaRef();
       const onInputHandler = jest.fn();
 
       const {
         result: { current: modifiedProps },
       } = renderHook(() =>
         useAutoExpand(ref, {
-          label: 'test',
+          ...baseTextareaProps,
           rows: 'auto',
           onInput: onInputHandler,
         }),
       );
 
       expect(modifiedProps.onInput).not.toEqual(onInputHandler);
       // We need to apply those props on a second textarea for code coverage.
-      render(<textarea aria-label="second" {...modifiedProps} />);
+      render(<TextArea aria-label="second" {...modifiedProps} />);
```

Comment:
```suggestion
      render(<TextArea label="second" {...modifiedProps} />);
```

Reply:
Makes sense! Passed after `modifiedProps` since these also contain a label (for the first textarea). We want to override them.

[f155d81](https://github.com/sumup-oss/circuit-ui/pull/1605/commits/f155d8141153ae3be223b9a7297f8554441790c4)